tools/wheel_resolver/src/wheel_tags/tags.py

```diff
@@ -15,16 +15,25 @@ def is_compatible(wheel, archs):
     current platform and python interpreter.
     Compatibility is based on https://www.python.org/dev/peps/pep-0425/
     """
+
     # Get the tag from the wheel we're checking
-    _, _, _, tag = parse_wheel_filename(wheel)
+    w = parse_wheel_filename(wheel)
 
     # taglist is a list of tags that we've got either from the user
     # or we've auto-generated them for this system
     taglist = generate_tags_from_all_archs(archs)
+    # TODO: Could probably cache these? ^^
+
+    if taglist is None:
+        logging.critical("No tags generated")
+    # else:
+    #     len_taglist = sum(1 for _ in taglist)
+    #     logging.critical('Generated %s tags', len_taglist)
```

Comment:
Did you mean to make these changes? Looks like it mgith be left over from another PR.